### DISCUSSION TITLE
Ability to select once daily, twice daily, three times, or four times.

### DISCUSSION THREAD

**Discussion by caj411** [diegopvlk/Dosage#50] [Ideas]
Many medications require multiple dosages daily on a schedule and an option for this would be nice feature.

**Comment by diegopvlk** [comment]
I don't understand this one, there is already multiple doses with each one with it's own time.
![image](https://github.com/diegopvlk/Dosage/assets/25941156/a66fd07b-764a-42f1-9614-042c68e14b9f)

**Comment by caj411** [comment]
I currently am putting in my notes, "once daily" or "twice daily", so its basically the directions for taking the medication that is on the pill bottle and I can see them 'at a glance'.  Maybe I should just use 'notes' for 'directions', its basically the same thing.  
I use the app as more of a database of all the medications and directions for taking them, not necessarily a reminder app. Its hard to keep up with a lot of medications and this app really does help me look at them "at a glance".  It actually has already helped me identify some incorrect instructions I had for my meds.
